scripts/dataset_processing/ljspeech/create_manifests_and_textfiles.py

```diff
@@ -22,48 +22,78 @@
 
 import sox
 import wget
+from nemo_text_processing.text_normalization.normalize import Normalizer
 
 from nemo.collections.common.parts.preprocessing import parsers
 
 parser = argparse.ArgumentParser()
-parser.add_argument('--ljspeech_base', required=True, default=None, type=str)
+parser.add_argument('--ljspeech_dir', required=True, type=str)
+parser.add_argument('--normalizer_class', choices=["ENCharParser", "Normalizer"], default="Normalizer", type=str)
+parser.add_argument('--whitelist_path', type=str)
+parser.add_argument('--save_transcripts_in_txt', action='store_true', default=False)
+parser.add_argument(
+    '--manifest_text_var_is_normalized',
+    action='store_true',
+    default=False,
+    help="If specified, the text in the manifest will contain normalized text. Otherwise, the text will contain the original text.",
+)
+
 args = parser.parse_args()
 
 
 def main():
-    filelist_base = 'https://raw.githubusercontent.com/NVIDIA/tacotron2/master/filelists/'
+    filelist_base = 'https://raw.githubusercontent.com/NVIDIA/tacotron2/master/filelists'
     filelists = ['train', 'val', 'test']
 
-    # NeMo parser for text normalization
-    text_parser = parsers.make_parser(name='en')
+    if args.normalizer_class == "ENCharParser":
+        normalizer_call = parsers.make_parser(name='en')._normalize
+    elif args.normalizer_class == "Normalizer":
+        if args.whitelist_path is None:
+            raise ValueError(
+                "whitelist_path is required when using Normalizer. "
+                "You can download the whitelist for LJSpeech from https://github.com/NVIDIA/NeMo/blob/main/nemo_text_processing/text_normalization/en/data/whitelist_lj_speech.tsv."
+            )
```

Comment:
Why not just automatically download it if not specified and save it to a cache dir?

Reply:
Done.